index.js

```diff
@@ -0,0 +1,144 @@
+const searchInput = document.getElementById("search__input");
+const searchButton = document.getElementById("search__btn");
+const cityLocation = document.getElementById("location");
+const humidity = document.getElementById("humidity");
+const weatherDescription = document.getElementById("description");
+const locationError = document.querySelector(".location__error");
+const weatherbox = document.getElementById("weather__box");
+const temprature = document.getElementById("temp");
+const wind = document.getElementById("wind");
+const loader = document.getElementById("loader");
+const weatherImg = document.getElementById("weatherImg");
+const errorMessage = document.getElementById("error");
+
+// API key and base URL for OpenWeatherMap API
+const API_KEY = "82005d27a116c2880c8f0fcb866998a0";
+const baseUrl = "http://api.openweathermap.org/data/2.5/weather?";
+
+// Function to update the UI with weather details
+const getWeatherDetails = (weatherinfo) => {
```

Comment:
This function is showing weather details on ui so it should be like showWeatherDetails